google/cloud/spanner_dbapi/connection.py

```diff
@@ -63,28 +69,64 @@ def __handle_update_ddl(self, ddl_statements):
         """
         Run the list of Data Definition Language (DDL) statements on the underlying
         database. Each DDL statement MUST NOT contain a semicolon.
-        Args:
-            ddl_statements: a list of DDL statements, each without a semicolon.
-        Returns:
-            google.api_core.operation.Operation.result()
+
+        :type ddl_statements: list
+        :param ddl_statements: A list of DDL statements, each without a
+                               semicolon.
         """
         self._raise_if_closed()
         # Synchronously wait on the operation's completion.
         return self.database.update_ddl(ddl_statements).result()
 
     def read_snapshot(self):
+        """Return a Snapshot of the linked Database.
+
+        :rtype: :class:`~google.cloud.spanner_v1.snapshot.Snapshot`
+        :returns: A snapshot of the linked Database.
+        """
         self._raise_if_closed()
         return self.database.snapshot()
 
     def in_transaction(self, fn, *args, **kwargs):
+        """Perform a unit of work in a linked Transaction, retrying on abort.
```

Comment:
Just a Transaction, not a linked one. This method in fact initiates a transaction and commits/rollbacks it, so there is no anything linked with it